test/unit/middlewares/authorizeBot.test.js

```diff
@@ -0,0 +1,52 @@
+const authorizeBot = require("../../../middlewares/authorizeBot");
+const sinon = require("sinon");
```

Comment:
Why are we using sinon?

Reply:
Sinon helps us for spying, mocking and stubbing. We are using it here for spying purposes to check the middleware output based on the inputs.

We can use a custom function as well, but that generally doesn't handle the cases and it is preinstalled so thought of using it

Reply:
ok